tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -620,4 +620,35 @@ public function userMountsShareOfferedByFromSpaceUsingTheGraphApi(string $user,
 		);
 		$this->featureContext->setResponse($response);
 	}
+
+	/**
+	 * @Then /^user "([^"]*)" should have sync (enabled|disabled) for share "([^"]*)"$/
+	 *
+	 * @param string $user
+	 * @param string $status
+	 * @param string $resource
+	 *
+	 * @return void
+	 * @throws \GuzzleHttp\Exception\GuzzleException
+	 */
+	public function userShouldHaveSyncEnabledOrDisabledForShare(string $user, string $status, string $resource):void {
+		$response = GraphHelper::getSharesSharedWithMe(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$user,
+			$this->featureContext->getPasswordForUser($user)
+		);
+		$responseBody = $this->featureContext->getJsonDecodedResponse($response);
+		$syncProperty = $status === 'enabled';
+		foreach ($responseBody["value"] as $value) {
+			if ($value["remoteItem"]["name"] === $resource) {
+				$propertyValue = $value["@client.synchronize"];
+			}
```

Comment:
break after getting the status
```suggestion
			if ($value["remoteItem"]["name"] === $resource) {
				$propertyValue = $value["@client.synchronize"];
				break;
			}
```